engines/router/missionctl/log/resultlog/bigquery.go

```diff
@@ -14,103 +13,65 @@ import (
 	"context"
 	"encoding/json"
 	"strings"
-	"time"
 
 	"cloud.google.com/go/bigquery"
+	"go.einride.tech/protobuf-bigquery/encoding/protobq"
```

Comment:
This custom library has more functionality for schema inference from the Protobuf struct (for example, conversion of proto maps -> BQ records). See: https://github.com/googleapis/google-cloud-go/issues/1119